diff --git a/core/lib/Drupal/Core/Utility/ThemeRegistry.php b/core/lib/Drupal/Core/Utility/ThemeRegistry.php
index bdc8188b5e4..7fb38d37707 100644
--- a/core/lib/Drupal/Core/Utility/ThemeRegistry.php
+++ b/core/lib/Drupal/Core/Utility/ThemeRegistry.php
@@ -190,7 +190,7 @@ public function getPreprocessInvokes(): array {
    * @internal
    */
   public function getGlobalPreprocess(): array {
-    return $this->get('global preprocess');
+    return $this->get('global preprocess') ?? [];
   }

 }
